client/__tests__/components/payment/stripe.test.ts

```diff
@@ -37,3 +42,14 @@ test('Uses Australian Stripe key for Australian delivery address', () => {
 		window.guardian.stripeKeyAustralia?.default,
 	);
 });
+
+test('Uses United States Stripe key for United States delivery address', () => {
+	const stripePublicKey = getStripeKey(
+		guardianWeeklySubscriptionUnitedStates.deliveryAddress?.country,
```

Comment:
why create the whole `guardianWeeklySubscriptionUnitedStates` object only to pick out just the delivery country?

Reply:
Thanks John - I have refactored here: https://github.com/guardian/manage-frontend/pull/1371/commits/28d83658f3cf49cd1e7ec472e3547d04d18f3649